chcemvediet/apps/inforequests/tests/test_views/test_attachments.py

```diff
@@ -88,11 +85,11 @@ def test_allowed_http_methods(self):
         self._login_user()
         attachment = self._create_attachment()
         url = reverse(u'inforequests:download_attachment', args=(attachment.pk,))
-
         allowed = [u'HEAD', u'GET']
         self.assert_allowed_http_methods(allowed, url)
 
-    def test_anonymous_user_gets_403_firbidden(self):
+    @unittest.skip(u'FIXME')
```

Comment:
@martinmacko47 Vyhodila sa podmienka `@login_required` preto nam view vrati 404. Avsak ak ja odhlasim usera, tak sa flushne session, a tym sa vymazu Attachmenty ktore som vytvoril `Attachment.objects.all()` vrati prazdne pole. Takze view padnem na riadku `attachment = Attachment.objects.get_or_404(pk=attachment_pk)` co nechceme. My chceme aby to padlo na podmienke `condition = permitted[attached_to_class]`, t.j. preto ze inforequest nie je published a profile anonymize_inforequest=True.

Reply:
@viliambalaz Ano, namiesto `self._logout_user()` treba ten attachment vytvorit pod jednym userom, toho usera nechat prihlaseneho, ale `.get(url)` spravit pod anonymnym userom. Teda tak, aby ten povodny user ostal stale prihlaseny. 

Nepamatam, ci sme uz mali niekde take testy, ze by sme simulovali pracu dvoch nezavislych userov. V principe potrebujeme nieco ako `self.client`, ale s dvomi sessions. Jedna session pre prihlaseneho usera, ktory vlastni attachment. A druha session pre anonymneho usera.

Analogicky pri tom druhom teste nizsie. Tam ta druha session bude namiesto anonymneho usera pre toho druheho prihlaseneho usera.

Reply:
:ok: